assets/playground/blueprint.json

```diff
@@ -17,7 +17,7 @@
 		"blogname": "AspireUpdate Demo Site"
 	},
 	"plugins": [
-		"https://github-proxy.com/proxy/?repo=AspirePress/AspireUpdate&branch=playground-ready",
+		"https://github-proxy.com/proxy/?repo=namithj/AspireUpdate&branch=main",
```

Comment:
Hold off on merging this PR until #173 is merged. Then merge `main` into this branch.

_Sorry for the disruption and inconvenience_

Reply:
#173 is merged.

Reply:
Great! The PR still needs work but at least there won't be extra headaches come merge time now that 173 is merged. 🙂